pages/models/regulations/governors/GovCT2/index.md

```diff
@@ -0,0 +1,566 @@
+> time constant Td means nearer ideal derivative. Note: The user has to
+> choose the derivative time constant according to the nature of the
+> input signal.
+
+*Documentation of the Modelica block*
+
+- The default value of $$T_\mathrm{D\,ratelim}=1 \mathrm{ms}$$ is used.
+- The same value is used for each ramp rate limiter block.
+
+### Input signals $$\omega$$ and $$P_\mathrm{ref}$$
+
+#### Power setpoint $$P_\mathrm{ref}$$
+
+Since the power feedback gets passed through the droop $$R$$, so for the
+power difference to make sense, the reference power needs to be
+multiplied by $$R$$ as well. This has historical reasons, see \[2\]
+section 2.3.3.5 (before digital times it used to be a mechanical lever
+that is moved, it did not matter what the lever positions meant).
+
+- **assumption:** In this implementation the reference power input needs
+  to be multiplied by $$R$$ before entering the governor model.
+
+The same is done in *DIgSILENT PowerFactory* GovCT2 at initialization:
+`inc(Pref) = R*Pfdbck + w`. ()
+
+#### Rotor speed $$\omega$$
+
+In *DIgSILENT PowerFactory*, different implementations for the speed
+signal are used. For example,
+
+- In the PSS/E compatible GGOV1 model, the reference speed is explicitly
+  subtracted from the measured speed to calculate `dw`, which is then
+  subtracted at the control error summation point. Here, the reference
+  power input is initialized to `inc(Pref) = R*Pfdbck+dw`.
+- In the GovCT2 model, the speed `w` is directly subtracted at the
+  control error summation point. Here, the reference power input is
+  initialized to `inc(Pref) = R*Pfdbck+w`.
+
+So in both cases the initial speed value that is subtracted from the
+control error summation point is – implicitly! – being added to that
+summation point through the `Pref` input. The effect is that in both
+cases $$\Delta \omega$$ is subtracted at the summation point.
+
+- **assumption:** In this model, this subtraction is being done
+  explicitly by calculating $$\Delta \omega$$ (see
+  <a href="#fig-modelSchema" class="quarto-xref">Figure 2</a>) and
+  **not** adding anything speed related to the power reference input.
+
+### Frequency-dependent limit
+
+The mapping between frequency and maximum valve position is implemented
+via a lookup table. The table consists of 10 pairs of values. Successive
+values need to decrease in frequency and max. power. If e.g. only 5
+pairs shall be used instead of all 10, only define pairs 1 to 5 and set
+all following pairs with fLim=0 and PLim = PLim5, resuling in a straight
+line in the lookup table. **This is implemented slightly different
+compared to \[1\] (p. 139).**
+
+### Governor output feedback delay
+
+To prevent an algebraic loop when using the governor output feedback, a
+first-order lag block with time constant $$T_\mathrm{last\,value}$$ has
+been added to the governor output feedback loop. The use of a unit delay
+block (1/z) has been found impractical.
+
+## Table of references & license
```

Comment:
the references numbers appears centered, is there a way to correct this?
if simpler, you can simply use bibtex references and add them in the references.bib and add just this piece of code there: {% bibliography --cited --file references  %}

Reply:
The literature links and citations are rendered automatically from a quarto file. This could be fixed manually, but here in this comment field, it renders properly. 

This code 
``` 
## Table of references & license

<div id="refs" class="references csl-bib-body" entry-spacing="0">

<div id="ref-iec61970-3022023" class="csl-entry">

<span class="csl-left-margin">\[1\]
</span><span class="csl-right-inline">IEC61970-302, “DIN EN IEC
61970-302 – Schnittstelle für Anwendungsprogramme für
Energiemanagementsysteme (EMS­API) Teil 302: Allgemeines
Informationsmodell (CIM) Dynamik.” Jun. 2023.</span>

</div>



</div>
```

renders into this: 

## Table of references & license

<div id="refs" class="references csl-bib-body" entry-spacing="0">

<div id="ref-iec61970-3022023" class="csl-entry">

<span class="csl-left-margin">\[1\]
</span><span class="csl-right-inline">IEC61970-302, “DIN EN IEC
61970-302 – Schnittstelle für Anwendungsprogramme für
Energiemanagementsysteme (EMS­API) Teil 302: Allgemeines
Informationsmodell (CIM) Dynamik.” Jun. 2023.</span>

</div>



</div>




Why is that not the case in the github pages? Are there settings to change how it gets rendered by pages?

Reply:
I fixed the issue